src/components/look-controls.js

```diff
@@ -259,12 +258,34 @@ module.exports.Component = registerComponent('look-controls', {
     }
   },
 
+  /**
+   * Shows grabbing cursor on scene
+   */
+
+  showGrabbingCursor: function () {
+    var currentStyle = this.el.sceneEl.canvas.getAttribute('style') || '';
```

Comment:
Just this:

```js
this.el.sceneEl.canvas.style.cursor = 'grabbing';
this.el.sceneEl.canvas.style.mozCursor = 'grabbing';
this.el.sceneEl.canvas.style.webkitCursor = 'grabbing';
```

Reply:
I'm afraid it won't work.. Take a look, what has to be prefixed is css property value, not property name. On the other hand, are You sure that those prefixes are needed? I tested it on latest chrome and firefox and it works without prefix, also take a look [here](https://caniuse.com/#search=grabbing).

Reply:
Yeah can just get rid of prefixes. It's not that big a feature anyways.